_topic_maps/_topic_map_rosa.yml

```diff
@@ -77,11 +77,24 @@ Topics:
   File: rosa-getting-support
 # - Name: Training for ROSA
 #   File: rosa-training
-#---
-#Name: Tutorials 
-#Dir: rosa_tutorials 
-#Distros: openshift-rosa
-#Topics:
+---
+Name: Tutorials 
+Dir: rosa_tutorials 
+Distros: openshift-rosa
+Topics:
+#- Name: Create a ROSA cluster using the CLI 
+#  File: rosa-mobb-cli-quickstart
+- Name: How to use AWS load balancer Operator on ROSA
+  File: rosa-mobb-aws-load-balancer-operator
+---
+Name: How to use ROSA
+Dir: rosa_how_to 
+Distros: openshift-rosa
+Topics:
+#- Name: Create a ROSA cluster using the CLI 
+#  File: rosa-mobb-cli-quickstart
+- Name: How to use AWS load balancer Operator on ROSA
```

Comment:
Looks the procedures is only available for ROSA STS and BYO VPC cluster. Please state it clearly.
For non-STS cluster we don't need to create the role via `aws iam` command

Reply:
Thank you! Will update ASAP!

Reply:
@paulczar do we have updated commands to make this procedure STS only?